EventListener/SetThemeEventListener.php

```diff
@@ -60,7 +60,10 @@ public function setActiveThemeOnRequest(RequestEvent $event): void
      */
     public function setActiveThemeOnPreviewPreRender(PreRenderEvent $event): void
     {
-        $theme = $this->themeRepository->findOneByName($event->getAttribute('webspace')->getTheme());
+        if (null === $themeName = $event->getAttribute('webspace')->getTheme()) {
```

Comment:
we try to avoid create variables inside if statements:

```suggestion
        $themeName = $event->getAttribute('webspace')->getTheme();
        if (null === $themeName) {
```